.circleci/scripts/binary_checkout.sh

```diff
@@ -62,6 +62,7 @@ popd
 
 # Clone the Builder master repo
 retry git clone -q https://github.com/pytorch/builder.git "$BUILDER_ROOT"
+git checkout release/1.9
```

Comment:
This pins PyTorch to release/1.9 rather than builder as one can see here:
https://app.circleci.com/pipelines/github/pytorch/pytorch/335990/workflows/a8aed0de-35d5-4ca5-9b0b-528b8914a37f/jobs/14110800?invite=true#step-103-225